packages/now-node/build.js

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env node
+const fs = require('fs-extra');
+const execa = require('execa');
+const { join } = require('path');
+
+async function main() {
+  const srcDir = join(__dirname, 'src');
+  const outDir = join(__dirname, 'dist');
+  const bridgeDir = join(__dirname, '../now-node-bridge');
+
+  // Copy shared dependencies
+  await Promise.all([
+    fs.copyFile(join(bridgeDir, 'src/bridge.ts'), join(srcDir, 'bridge.ts')),
+    fs.copyFile(
+      join(bridgeDir, 'src/launcher.ts'),
+      join(srcDir, 'launcher.ts')
+    ),
+  ]);
+
+  // Start fresh
+  await fs.remove(outDir);
+
+  // Build TypeScript files
+  await execa('tsc', [], {
+    stdio: 'inherit',
+  });
+
+  // Copy type file for ts test
+  await fs.copyFile(
+    join(outDir, 'types.d.ts'),
+    join(__dirname, 'test/fixtures/15-helpers/ts/types.d.ts')
+  );
+
+  // Setup symlink for symlink test
+  const symlinkTarget = join(__dirname, 'test/fixtures/11-symlinks/symlink');
+  await fs.remove(symlinkTarget);
+  await fs.symlink('symlinked-asset', symlinkTarget);
+
+  // Use types.d.ts as the main types export
+  await fs.rename(join(outDir, 'types.d.ts'), join(outDir, 'types'));
+  await Promise.all(
+    (await fs.readdir(outDir))
+      .filter(p => p.endsWith('.d.ts'))
+      .map(p => fs.remove(join(outDir, p)))
+  );
+  await fs.rename(join(outDir, 'types'), join(outDir, 'index.d.ts'));
```

Comment:
```suggestion
  await Promise.all(
    (await fs.readdir(outDir))
      .filter(p => p.endsWith('.d.ts') && p !== 'types.d.ts')
      .map(p => fs.remove(join(outDir, p)))
  );
  await fs.rename(join(outDir, 'types.d.ts'), join(outDir, 'index.d.ts'));
```

We can remove a step here since the first step (rename) was intended to prevent deleting in the second step. And the third step renamed again.

We can do the same in two steps now 🙂